src/main/java/org/apache/taglibs/standard/util/XmlUtil.java

```diff
@@ -69,7 +69,14 @@
     private static final DocumentBuilderFactory PARSER_FACTORY;
     private static final SAXTransformerFactory TRANSFORMER_FACTORY;
     private static final SAXParserFactory SAXPARSER_FACTORY;
+
+    private static boolean secureProcessing;
+
     static {
+
+        String secure = System.getProperty("org.apache.taglibs.secureXmlProcessing");
+        secureProcessing = ! (secure != null && secure.equalsIgnoreCase("false"));
```

Comment:
Ah now, i see what you are saying, variant with default value is not there.
Inconsistency, Integer.getInteger() has both variants. oh well :(